server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -258,7 +260,7 @@ export class MongoCollection<T> implements core.ICollection<T>, core.IRetryable
             InitialRetryIntervalInMs, // retryAfterMs
             (error: any, numRetries: number, retryAfterInterval: number) =>
                 numRetries * retryAfterInterval, // retryAfterIntervalCalculator
-            undefined, /* onErrorFn */
+            this.errorSanitizer, /* onErrorFn */
```

Comment:
Would it help to also add that to the runWithRetry used by scriptorium?

https://github.com/microsoft/FluidFramework/blob/df52eec6ac53116f0a455da2c887698dfc7bb704/server/routerlicious/packages/lambdas/src/scriptorium/lambda.ts#L175

Reply:
Not necessary. The runWithRetry in the Mongo facade is the innermost runWithRetry and having the errorSanitizer there makes sure that anything above that will get the sanitized version of error.